content/docs/0.9.x/reference/bridge/_index.md

```diff
@@ -172,12 +172,23 @@ and additional data is gathered from the client. For more information about this
 
 ### Settings
 
+#### Project settings
 In the settings view the project settings can be edited. 
 To get more information about how to manage your projects with the settings, visit [Manage Projects](https://keptn.sh/docs/0.9.x/reference/bridge/manage_projects).
 
 {{< popup_image
-link="./assets/settings.png"
-caption="Keptn Bridge Settings View">}}
+link="./assets/project-settings.png"
+caption="Keptn Bridge Project Settings View">}}
+
+
+#### Services settings
+In this view services can be created, updated and deleted.
+To get more information about how to manage your projects with the settings, visit [Manage Services](https://keptn.sh/docs/0.9.x/reference/bridge/manage_services).
```

Comment:
To get more information about how services can be managed in your settings, visit ...